spec/imports/imports.spec.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+var fs = require('fs-extra');
+var logger = require('log4js').getLogger('Plugins-e2e');
+var _ = require('lodash');
+var components = require('../../src/components');
+
+describe('imports section', function() {
+    describe('login', function() {
+        browser.get('/');
+
+        components.login.login('user-' + new Date().getTime());
+        // navigate to definitions tab
+        browser.sleep(1000);
+        logger.info('navigate to imports tab');
+        components.layout.goToImports();
+    });
+
+    describe('imports field', function() {
+        it('import url should exist', function(done) {
+            expect(components.imports.page.countElements(element.all(by.repeater('imports in globals.stencils.importsData')))).toBe(9);//default amount of imports
+            browser.sleep(200).then(done);
+        });
+        it('should open then close import modal view', function(done) {
+            components.imports.page.openImportsViewDialog();//click show modal with import data btn
+            browser.sleep(500);//waiting when modal appears
+            expect(components.imports.page.isElementDisplayed(element(by.css('.modal.fade.in')))).toBe(true);//modal should appear
+
+            components.modals.modal.cancelUsingSelector();//click cancel btn
```

Comment:
please move this specific sleep into the cancel function as we will always have to add it.